framework/src/errors.ts

```diff
@@ -23,8 +24,20 @@ export class FrameworkError extends Error {
 	}
 }
 
+export class NonceNotReadyError extends FrameworkError {
```

Comment:
Just to understand what's a "Nonce Not Ready" ?

Reply:
it suppose to be `nonce_too_high`, will update

Reply:
Ah but it should be bit more generic than too high, so that's why i put `nonce_not_ready` to be processed/included in a block

Reply:
maybe `NonceNotWithinLimits` (either lower or too high)? But ok to keep I guess I only have problem with the Ready word as it would seem nonce is an active component an it's only a prop. But as I said ok to keep.

Reply:
`NONCE_OUT_OF_BOUNDS`?

Reply:
yeah it doesnt sound assuring, maybe `nonSequentialNonceError` ?

Reply:
SequentialNonce is specific to the sequence module rule, so i wanted to have generic name where nonce is not ready to be included in a block, but allowed in the pool